taskvine/src/worker/vine_worker.c

```diff
@@ -1121,6 +1168,29 @@ static void disconnect_manager(struct link *manager)
 	}
 }
 
+void send_stdout(struct link *l, int64_t task_id)
+{
+	struct vine_process *p = itable_lookup(procs_table, task_id);
+	if (!p) {
+		send_message(l, "failure %" SCNd64 "\n", task_id);
```

Comment:
Please make this correspond in format to the `error` message used by `vine_transfer_put/get`.  Or even better, is it possible to use the same code?

Reply:
changed format to correspond with other error messages.